addons/io/org.openhab.io.homekit/src/main/java/org/openhab/io/homekit/internal/accessories/HomekitSecuritySystemImpl.java

```diff
@@ -0,0 +1,268 @@
+                    systemState = TargetSecuritySystemState.DISARMED;
+                } else {
+                    logger.error(
+                            "Unrecognized security system state: {}. Expected stay, away, night, or disarmed strings in value.",
+                            stringValue);
+                    systemState = TargetSecuritySystemState.DISARMED;
+                }
+            }
+        }
+
+        if (systemState == null) {
+            logger.error("Unable to get target security system state");
+        } else {
+            logger.debug("targetSecuritySystemState: {}", systemState);
+        }
+        return CompletableFuture.completedFuture(systemState);
+    }
+
+    @Override
+    public CompletableFuture<SecuritySystemAlarmType> getAlarmTypeState() {
+        SecuritySystemAlarmType alarmType = null;
+
+        if (alarmTypeItemName != null) {
+            Item item = getItemRegistry().get(alarmTypeItemName);
+            if (item != null) {
+                State itemState = item.getState();
+                String stringValue = itemState.toString();
+
+                if (stringValue.equals("noAlarm")) {
+                    alarmType = SecuritySystemAlarmType.CLEARED; // TODO: Update this constant:
+                                                                 // https://github.com/beowulfe/HAP-Java/pull/45
+                } else if (stringValue.equals("unknown")) {
+                    alarmType = SecuritySystemAlarmType.UNKNOWN;
+                } else if (stringValue.equals("UNDEF") || stringValue.equals("NULL")) {
+                    logger.debug(
+                            "Current security system alarm type not available. Relaying value of CLEARED to Homekit");
+                    alarmType = SecuritySystemAlarmType.CLEARED;
+                } else {
+                    logger.error("Unrecognized security system alarm type: {}. Expected noAlarm or unknown",
+                            stringValue);
+                    alarmType = SecuritySystemAlarmType.CLEARED;
+                }
+            }
+        }
+        if (alarmType == null) {
+            logger.error("Unable to get alarm type");
+        } else {
+            logger.debug("alarmType: {}", alarmType);
+        }
+        return CompletableFuture.completedFuture(alarmType);
+    }
+
+    @Override
+    public void setTargetSecuritySystemState(TargetSecuritySystemState state) throws Exception {
+        String stateString = null;
+        switch (state) {
+            case STAY_ARM:
+                stateString = "stayArm";
+                break;
+            case NIGHT_ARM:
+                stateString = "nightArm";
+                break;
+            case AWAY_ARM:
+                stateString = "awayArm";
+                break;
+            case DISARMED: // TODO: Update this constant: https://github.com/beowulfe/HAP-Java/pull/44
```

Comment:
It would be nice if @beowulfe creates a 1.1.5 release so these TODOs can be resolved. :-)